app_server/test/controllers/UserController.js

```diff
@@ -25,19 +25,43 @@ var bob = {
   description: 'bam bam bam'
 };
 
+var alice = {
+  username: 'Alice',
+  alias: 'Alice in the wonderland',
+  email: 'alice@apple.com',
+  password: 'generated',
+  accessToken: 'anaccesstoken',
+  platformType: 'facebook',
+  platformId: '45454545454',
+  description: 'nil'
+};
+
 lab.experiment('UserController Tests', function () {
   lab.beforeEach({timeout: 10000}, function (done) {
     TestUtils.resetDatabase(done);
   });
 
-  lab.test('Get list of users', function (done) {
+  lab.test('Get list of users valid empty', function (done) {
     Router.inject({url: '/api/users', credentials: testAccount},
       function (res) {
-        Code.expect(res.result).to.equal('Hello Sharmine!');
+        Code.expect(res.result).to.have.length(0);
         done();
       });
   });
 
+  lab.test('Get list of users valid default', function (done) {
+    Service.createNewUser(bob).then(function (result) {
+      return Service.createNewUser(alice);
+    }).then(function() {
```

Comment:
Right. Well, if you want to shorten that, `() => Service.createNewUser(alice)` is equivalent to `function () { return Service.createNewUser(alice); }`. The variable passed in (`result`) appears to be unused in this case.

(On a side note, do we just assume those two calls succeed and not bother testing them since they're covered elsewhere?)